### DISCUSSION TITLE
修改gradient checkpointing方法，进一步降低训练显存

### DISCUSSION THREAD

**Discussion by ninghongbo123** [zai-org/ChatGLM3#648] [Bad Case]
1、目前的gradient checkpointing是对每一层进行checkpoint，即28个checkpoint;
2、我想进一步降低显存，于是只对其中14个进行checkpoint，但是为什么显存需求更大导致oom？理论上会进一步降低显存才对呢？
3、可能是我修改的方式不对，或者理解不对？
请大佬指导一二。

**Comment by zRzRzRzRzRzRzR** [comment]
你可以到 #253 问问？我没有太看懂
